course/templates/course/index.html

```diff
@@ -12,85 +12,148 @@ <h1>{{ welcome_text|safe }}</h1>
 </div>
 {% endif %}
 
-<div class="row course-thumbnails">
-    {% site_advert as advert %}
-    {% if advert %}
+{% if my_instances %}
+<div class="frontpage-panel">
+    <h2 class="panel-heading">{% trans "My courses" %}</h2>
+    <div class="row frontpage">
+        {% for instance in my_instances %}
+        {% with instance_url=instance|url %}
         <div class="col-sm-6 col-md-4 col-lg-3">
-            <div class="thumbnail advert-thumbnail">
-                <div class="image" aria-hidden="true">
-                    {% if advert.image %}
-                        <img src="{{ advert.image }}" alt="">
-                    {% else %}
-                        <h1 class="glyphicon glyphicon-comment"></h1>
-                    {% endif %}
+            <div class="frontpage-course card">
+                <a href="{{ instance_url }}">
+                    <div class="course-image" aria-hidden="true">
+                        {% if instance.image %}
+                        <img class="card-img-top" src="{{ instance.image.url }}" alt="" />
+                        {% else %}
+                        <div class="card-img-top">
+                            <i class="glyphicon glyphicon-book"></i>
+                        </div>
+                        {% endif %}
+                    </div>
+                </a>
+                <div class="card-body">
+                    <h3 class="card-title" lang="{{ instance.language }}">
+                        <a href="{{ instance_url }}">
+                            {{ instance.course.name|parse_localization }}<br />
+                        </a>
+                    </h3>
+                    <h4 class="card-subtitle" lang="{{ instance.language }}">
```

Comment:
Only just spotted this one. Have you considered that this may work better if not marked up as a heading. It looks good visually, but I don't think it's behaving like a subheading to the `h3` above it.

Reply:
In my mind it actually kind of makes sense, there are several instances of the same course. So a sort of "sub-heading". And it looked good style-wise :) But hmm.. I guess there is a point, I could just change it to <p>-tag and style it to look like it does now

Reply:
This has been changed.